src/core/components/SideMenuSection/index.js

```diff
@@ -5,18 +5,28 @@ import styled from 'styled-components';
 // TODO - confirm colour with design
 const SectionContainer = styled.div`
   border-top: 1px solid #f7f7f8;
-  padding: 0 8px;
+  padding: 0.5rem 0 0.5em;
 `;
 
-const ListHeading = styled.h4`
+export const ListHeading = styled.h4`
   ${({ theme }) => theme.typography.title};
-  padding: 0 8px;
-  margin: 1em 0;
+  padding: 0.5rem 0.8rem;
+  margin: 0 0 0.5rem;
+  border-bottom: 1px solid #f7f7f8;
+  flex-direction: row;
+  align-items: center;
+  display: inline-flex;
+  width: 100%;
+  font-size: 0.9rem !important;
```

Comment:
can we avoid this !important somehow? I remember I had issues with this in the past

Reply:
Not atm unfortunatly. Amity messed up something with style composition, so without important, it gets overwritten from somewhere else. Which should not get a thing. Will be fixed once we start replacing Amity components.